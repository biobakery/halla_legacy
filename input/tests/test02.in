sample	SRS050184	SRS016516	SRS052330	SRS011355	SRS019600	SRS024428	SRS052756	SRS047335	SRS063417	SRS016559	SRS015073	SRS016191	SRS015425	SRS015225	SRS064376	SRS064704	SRS052620	SRS014629	SRS023850	SRS042858	SRS051505	SRS049237	SRS062520	SRS014494	SRS056695	SRS020349	SRS013542	SRS065347	SRS019379	SRS054962	SRS015168	SRS017497	SRS022092	SRS015054	SRS022158	SRS024068	SRS012294	SRS057807	SRS022734	SRS018769	SRS077751	SRS017520	SRS042428	SRS055298	SRS011584	SRS017700	SRS011269	SRS021969	SRS019989	SRS016111	SRS044742
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales	0.0046188	0	0	6.92e-05	0.0003947	0.0016521	0.0010791	0	0	8.52e-05	0	0.0001012	0	0.0003522	0.0005802	0.000425	0.0038839	0	0.0025439	0	0.0009873	0	0	0	2.94e-05	5.76e-05	0.0024253	0	0	0.0131237	9.4e-05	0.0014775	0	9.44e-05	0	0.0014245	0	0.0001014	0	0	0	0.0002424	2.43e-05	3.94e-05	0	0	0.0001173	0	0.0001189	0	0
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales|f__Actinomycetaceae|g__Actinomyces	0.0001306	0	0	0	0	0	0	0	0	0	0	0	0	0	0	8.08e-05	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales|f__Corynebacteriaceae|g__Corynebacterium|s__Corynebacterium_amycolatum	0	0	0	0	0	0	0	0	0	0	0	1e-05	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales|f__Mycobacteriaceae|g__Mycobacterium	0	0	0	0	0	0.001271	0	0	0	0	0	0	0	0.0003522	0	0	0.0038839	0	0.0018112	0	0.000367	0	0	0	0	0	0.0024253	0	0	0	9.4e-05	0.0014775	0	9.44e-05	0	0.0014245	0	0	0	0	0	0	0	0	0	0	0.0001173	0	0.0001189	0	0
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales|f__Mycobacteriaceae|g__Mycobacterium|s__Mycobacterium_abscessus	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	9.44e-05	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Bifidobacteriales|f__Bifidobacteriaceae|g__Bifidobacterium	0	0	0	0	0	0.0042791	0	0	0	0	0.0005275	0	0	0	0	0	0.416292	0	0.135502	0	0.0033262	0	0	0	0	0	0.0197996	0	0	4.66e-05	0	0.0047989	0	0	0	0.0018442	0	0	0.0391844	0	0	0	0	0	0	0	0.0015441	0	0.0001025	0	0
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Coriobacteriales|f__Coriobacteriaceae|g__Atopobium	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0074239	0	0.0071804	0	0	0	0	0	0.181252	0	0	0	0	0.265973	0	0	0	0.0015031	0	0	0	0	0	0	0	0	0	0	0.0041648	0	5.27e-05	0	0
k__Bacteria|p__Bacteroidetes|c__Bacteroidia|o__Bacteroidales|f__Bacteroidaceae|g__Bacteroides|s__Bacteroides_cellulosilyticus	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0004995	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
k__Bacteria|p__Bacteroidetes|c__Bacteroidia|o__Bacteroidales|f__Bacteroidaceae|g__Bacteroides|s__Bacteroides_ovatus	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0008368	0	0	0.0005057	0	0.0038247	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
k__Bacteria|p__Bacteroidetes|c__Bacteroidia|o__Bacteroidales|f__Bacteroidaceae|g__Bacteroides|s__Bacteroides_plebeius	0	0	0	0	0	0	0	0	0	0	0	0	0	0	8.64e-05	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
k__Bacteria|p__Bacteroidetes|c__Bacteroidia|o__Bacteroidales|f__Bacteroidaceae|g__Bacteroides|s__Bacteroides_stercoris	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.000162	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
k__Bacteria|p__Bacteroidetes|c__Bacteroidia|o__Bacteroidales|f__Porphyromonadaceae|g__Parabacteroides|s__Parabacteroides_merdae	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0003887	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0001445	0	0	0	0	0	0	0	0	0	0	0
k__Bacteria|p__Bacteroidetes|c__Bacteroidia|o__Bacteroidales|f__Rikenellaceae|g__Alistipes|s__Alistipes_unclassified	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.001702	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
k__Bacteria|p__Bacteroidetes|c__Flavobacteria|o__Flavobacteriales|f__Flavobacteriaceae	0.0008695	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
k__Bacteria|p__Bacteroidetes|c__Flavobacteria|o__Flavobacteriales|f__Flavobacteriaceae|g__Capnocytophaga	0.0008695	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
k__Bacteria|p__Firmicutes|c__Bacilli|o__Bacillales	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0025691	0	0	0	0	0	0	0	0	0	0	0	0.0003094	0	0	0	0	0	0	0	0	0
k__Bacteria|p__Firmicutes|c__Bacilli|o__Lactobacillales	0.992299	1	1	0.999696	0.999605	0.663081	0.978293	0.987621	0.997237	0.999915	0.984792	0.998338	0.999207	0.98051	0.997984	0.999219	0.579824	0.998983	0.720498	1	0.803619	0.970992	1	0.952881	0.999866	0.997153	0.014053	0.998049	0.977727	0.968664	0.995309	0.0010376	1	0.989373	0.989161	0.91637	1	0.99977	0.960816	0.998025	1	0.998543	0.960849	0.957963	0.998733	0.999426	0.876182	0.998509	0.988527	0.998265	0.943673
k__Bacteria|p__Firmicutes|c__Bacilli|o__Lactobacillales|f__Enterococcaceae|g__Enterococcus|s__Enterococcus_faecalis	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0002337	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
k__Bacteria|p__Firmicutes|c__Bacilli|o__Lactobacillales|f__Lactobacillaceae	0.992299	1	0.998409	0.999696	0.999605	0.663081	0.978293	0.987621	0.997237	0.999915	0.984792	0.998338	0.999207	0.98051	0.997984	0.999219	0.579824	0.99833	0.717974	0.999734	0.803619	0.970992	1	0.952881	0.999866	0.997153	0.014053	0.998049	0.977727	0.968492	0.995309	0.0010376	1	0.989373	0.989161	0.91637	1	0.99977	0.960816	0.998025	1	0.998543	0.960849	0.957721	0.998733	0.999426	0.871594	0.998509	0.988527	0.998265	0.943673
k__Bacteria|p__Firmicutes|c__Bacilli|o__Lactobacillales|f__Lactobacillaceae|g__Lactobacillus|s__Lactobacillus_crispatus	0	1	0	0.997322	0.999605	0	0	0	0	0.999915	0.289139	0.993281	0.999207	0	0.996325	0	0	0	0	0.999734	0.259984	0.580832	0	0.693682	0.98916	0.997153	0	0.961317	0	0.0012135	0.982164	0	0	0.960478	0.934079	0	0	0.994321	0	0.996157	1	0.983034	0	0	0	0.995659	0	0.949207	0.980375	0	0.563946
k__Bacteria|p__Firmicutes|c__Bacilli|o__Lactobacillales|f__Lactobacillaceae|g__Lactobacillus|s__Lactobacillus_jensenii	0.284115	0	0.986216	0.0009156	0	0	0	0.0142089	0.997237	0	0.0025304	0.0044751	0	0.0737295	0	0.37807	0	0.99833	0	0	0.0223026	0.017491	0	0.0569356	0.0103512	0	0	0.0367316	0	0.004259	0.0023563	0	1	0.0288951	0.049632	0.91637	1	0.0054493	0	0	0	0.0152906	0.266462	0.0120394	0.998733	0.0034226	0.831625	0	0.0077163	0	0.0218967
k__Bacteria|p__Firmicutes|c__Bacilli|o__Lactobacillales|f__Streptococcaceae|g__Streptococcus|s__Streptococcus_mitis	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	3.22e-05	0	0	0	0	0	0	0	0	0	0.0001724	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0001434	0	0	0	0	0	0	0
k__Bacteria|p__Firmicutes|c__Clostridia|o__Clostridiales	0.0001935	0	0	0.0001816	0	0.0003571	0.0154824	0.011938	0	0	0	0.0009027	0	0.0136875	0	0	0	0.0010165	0	0	0.0066801	0.0001491	0	0.0141559	0.0001049	0.0004807	0.0054738	9.75e-05	0	0.0153617	0.0021523	0.005075	0	0.000205	0	0	0	0.0001094	0	0.0005879	0	0	0.0387202	0.0120801	0	0.0001329	0.0106956	0.0014907	0	0	0.0173023
k__Bacteria|p__Firmicutes|c__Clostridia|o__Clostridiales|f__Clostridiales_Family_XI_Incertae_Sedis|g__Anaerococcus	0	0	0	2.05e-05	0	0	0	0.000177	0	0	0	0.0002854	0	0.0003118	0	0	0	0	0	0	0.0008413	0	0	0	0	0	0.0021819	0	0	0.0001519	0.0014213	0.0005081	0	0.000116	0	0	0	0	0	0	0	0	0	3.56e-05	0	5.63e-05	0.0029224	0	0	0	0.0005169
k__Bacteria|p__Firmicutes|c__Clostridia|o__Clostridiales|f__Clostridiales_Family_XI_Incertae_Sedis|g__Anaerococcus|s__Anaerococcus_lactolyticus	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0001287	0	0	0	0
k__Bacteria|p__Firmicutes|c__Clostridia|o__Clostridiales|f__Clostridiales_Family_XI_Incertae_Sedis|g__Anaerococcus|s__Anaerococcus_tetradius	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0021819	0	0	0	0.0002199	0.0004406	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
k__Bacteria|p__Firmicutes|c__Clostridia|o__Clostridiales|f__Clostridiales_Family_XI_Incertae_Sedis|g__Anaerococcus|s__Anaerococcus_vaginalis	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	4.81e-05	0	0	0	0
k__Bacteria|p__Firmicutes|c__Clostridia|o__Clostridiales|f__Peptostreptococcaceae|g__Peptostreptococcus	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.001547	0	0	0	0	0	0	0	0	0	0	0	0	0	0	5.51e-05	0	0	0	0
k__Bacteria|p__Proteobacteria	0.0006886	0	0	0	0	0.0007347	0	0	0.0027634	0	0	0.0002536	0.0007929	0.005343	4.29e-05	9.25e-05	0	0	0	0	0	0	0	0	0	0.0002337	0	0	0	0.0002348	0	0	0	0.0035161	0	0	0	0	0	0	0	0.0009054	0.0004067	0	0	4.95e-05	0	0	0.00069	0.0017349	0
k__Bacteria|p__Proteobacteria|c__Betaproteobacteria	0.0006886	0	0	0	0	0.0007347	0	0	0	0	0	7.99e-05	0	0	4.29e-05	9.25e-05	0	0	0	0	0	0	0	0	0	0.0002337	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	4.95e-05	0	0	0.00069	0	0
k__Bacteria|p__Proteobacteria|c__Betaproteobacteria|o__Burkholderiales	0	0	0	0	0	0.0007347	0	0	0	0	0	1.01e-05	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0002337	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	4.95e-05	0	0	0.00069	0	0
k__Bacteria|p__Proteobacteria|c__Betaproteobacteria|o__Burkholderiales|f__Burkholderiaceae|g__Burkholderia|s__Burkholderia_unclassified	0	0	0	0	0	0.0007185	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0002091	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	4.95e-05	0	0	0.0005873	0	0
k__Bacteria|p__Proteobacteria|c__Betaproteobacteria|o__Neisseriales|f__Neisseriaceae|g__Neisseria|s__Neisseria_sicca	0.0001561	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
k__Bacteria|p__Proteobacteria|c__Gammaproteobacteria|o__Pseudomonadales	0	0	0	0	0	0	0	0	0.0027634	0	0	0.0001737	0.0007929	0.005343	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0034774	0	0	0	0	0	0	0	0	0.0004067	0	0	0	0	0	0	0.0017349	0
k__Bacteria|p__Proteobacteria|c__Gammaproteobacteria|o__Pseudomonadales|f__Pseudomonadaceae|g__Pseudomonas	0	0	0	0	0	0	0	0	0.0027634	0	0	0.0001737	0.0007929	0.0052871	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0034774	0	0	0	0	0	0	0	0	0.0004067	0	0	0	0	0	0	0.0017349	0
k__Bacteria|p__Tenericutes|c__Mollicutes|o__Mycoplasmatales|f__Mycoplasmataceae|g__Ureaplasma	0.0011963	0	0	0	0	0.0414583	0.0049846	0.0003345	0	0	0.0007801	0	0	0	0	0	0	0	0	0	0.0010529	0.0279891	0	0	0	0	0.001845	0.0018535	0	0	0.0012679	0	0	0.0068117	0.0108388	0	0	0	0	0	0	0	0	0.0298654	0	0	0	0	0.0008666	0	0.0059283
k__Bacteria|p__Tenericutes|c__Mollicutes|o__Mycoplasmatales|f__Mycoplasmataceae|g__Ureaplasma|s__Ureaplasma_unclassified	0	0	0	0	0	0.0115082	0.0029732	0	0	0	0.0005128	0	0	0	0	0	0	0	0	0	0.0006891	0.005426	0	0	0	0	0.0009243	0	0	0	0.0011457	0	0	0.0005154	0.0044096	0	0	0	0	0	0	0	0	0.0037195	0	0	0	0	0.000208	0	0
k__Bacteria|p__Verrucomicrobia|c__Verrucomicrobiae|o__Verrucomicrobiales|f__Verrucomicrobiaceae|g__Akkermansia|s__Akkermansia_muciniphila	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1.73e-05	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
