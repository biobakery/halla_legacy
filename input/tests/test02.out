Clusters
k__Bacteria|p__Firmicutes|c__Bacilli|o__Lactobacillales|f__Enterococcaceae|g__Enterococcus|s__Enterococcus_faecalis	k__Bacteria|p__Firmicutes|c__Bacilli|o__Lactobacillales|f__Streptococcaceae|g__Streptococcus|s__Streptococcus_mitis
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales|f__Corynebacteriaceae|g__Corynebacterium|s__Corynebacterium_amycolatum	k__Bacteria|p__Proteobacteria|c__Gammaproteobacteria|o__Pseudomonadales	k__Bacteria|p__Proteobacteria|c__Gammaproteobacteria|o__Pseudomonadales|f__Pseudomonadaceae|g__Pseudomonas
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales|f__Actinomycetaceae|g__Actinomyces	k__Bacteria|p__Bacteroidetes|c__Flavobacteria|o__Flavobacteriales|f__Flavobacteriaceae	k__Bacteria|p__Bacteroidetes|c__Flavobacteria|o__Flavobacteriales|f__Flavobacteriaceae|g__Capnocytophaga	k__Bacteria|p__Proteobacteria|c__Betaproteobacteria|o__Neisseriales|f__Neisseriaceae|g__Neisseria|s__Neisseria_sicca
k__Bacteria|p__Bacteroidetes|c__Bacteroidia|o__Bacteroidales|f__Bacteroidaceae|g__Bacteroides|s__Bacteroides_cellulosilyticus	k__Bacteria|p__Bacteroidetes|c__Bacteroidia|o__Bacteroidales|f__Bacteroidaceae|g__Bacteroides|s__Bacteroides_ovatus	k__Bacteria|p__Bacteroidetes|c__Bacteroidia|o__Bacteroidales|f__Porphyromonadaceae|g__Parabacteroides|s__Parabacteroides_merdae	k__Bacteria|p__Bacteroidetes|c__Bacteroidia|o__Bacteroidales|f__Rikenellaceae|g__Alistipes|s__Alistipes_unclassified	k__Bacteria|p__Verrucomicrobia|c__Verrucomicrobiae|o__Verrucomicrobiales|f__Verrucomicrobiaceae|g__Akkermansia|s__Akkermansia_muciniphila
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales|f__Mycobacteriaceae|g__Mycobacterium|s__Mycobacterium_abscessus	k__Bacteria|p__Firmicutes|c__Clostridia|o__Clostridiales|f__Clostridiales_Family_XI_Incertae_Sedis|g__Anaerococcus
k__Bacteria|p__Tenericutes|c__Mollicutes|o__Mycoplasmatales|f__Mycoplasmataceae|g__Ureaplasma	k__Bacteria|p__Tenericutes|c__Mollicutes|o__Mycoplasmatales|f__Mycoplasmataceae|g__Ureaplasma|s__Ureaplasma_unclassified
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales	k__Bacteria|p__Firmicutes|c__Bacilli|o__Lactobacillales	k__Bacteria|p__Firmicutes|c__Bacilli|o__Lactobacillales|f__Lactobacillaceae	k__Bacteria|p__Firmicutes|c__Bacilli|o__Lactobacillales|f__Lactobacillaceae|g__Lactobacillus|s__Lactobacillus_crispatus	k__Bacteria|p__Firmicutes|c__Bacilli|o__Lactobacillales|f__Lactobacillaceae|g__Lactobacillus|s__Lactobacillus_jensenii	k__Bacteria|p__Firmicutes|c__Clostridia|o__Clostridiales
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales|f__Mycobacteriaceae|g__Mycobacterium	k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Bifidobacteriales|f__Bifidobacteriaceae|g__Bifidobacterium	k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Coriobacteriales|f__Coriobacteriaceae|g__Atopobium	k__Bacteria|p__Firmicutes|c__Clostridia|o__Clostridiales|f__Clostridiales_Family_XI_Incertae_Sedis|g__Anaerococcus|s__Anaerococcus_lactolyticus	k__Bacteria|p__Firmicutes|c__Clostridia|o__Clostridiales|f__Clostridiales_Family_XI_Incertae_Sedis|g__Anaerococcus|s__Anaerococcus_vaginalis	k__Bacteria|p__Firmicutes|c__Clostridia|o__Clostridiales|f__Peptostreptococcaceae|g__Peptostreptococcus
k__Bacteria|p__Proteobacteria	k__Bacteria|p__Proteobacteria|c__Betaproteobacteria
k__Bacteria|p__Proteobacteria|c__Betaproteobacteria|o__Burkholderiales	k__Bacteria|p__Proteobacteria|c__Betaproteobacteria|o__Burkholderiales|f__Burkholderiaceae|g__Burkholderia|s__Burkholderia_unclassified
One	Two	MI	MID	Pperm	Pboot	P
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales|f__Corynebacteriaceae|g__Corynebacterium|s__Corynebacterium_amycolatum	k__Bacteria|p__Proteobacteria|c__Betaproteobacteria|o__Burkholderiales	0.139233	0.860767	0.0833333	5.47091e-05	0.0833333
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales|f__Actinomycetaceae|g__Actinomyces	k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales	0.11548	0.92714	0.321429	0.000971104	0.321429
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales|f__Actinomycetaceae|g__Actinomyces	k__Bacteria|p__Proteobacteria	0.207123	0.86932	0.047619	3.18826e-15	0.047619
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales|f__Mycobacteriaceae|g__Mycobacterium|s__Mycobacterium_abscessus	k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales|f__Mycobacteriaceae|g__Mycobacterium	0.139233	0.860767	0.0595238	1.50369e-05	0.0595238
k__Bacteria|p__Tenericutes|c__Mollicutes|o__Mycoplasmatales|f__Mycoplasmataceae|g__Ureaplasma	k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales	0.249204	0.911232	0.880952	1.68763e-21	0.880952
k__Bacteria|p__Tenericutes|c__Mollicutes|o__Mycoplasmatales|f__Mycoplasmataceae|g__Ureaplasma	k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales|f__Mycobacteriaceae|g__Mycobacterium	0.363904	0.870375	0.119048	5.64627e-29	0.119048
k__Bacteria|p__Tenericutes|c__Mollicutes|o__Mycoplasmatales|f__Mycoplasmataceae|g__Ureaplasma	k__Bacteria|p__Proteobacteria	0.186412	0.933599	0.833333	1.13323e-05	0.833333
k__Bacteria|p__Tenericutes|c__Mollicutes|o__Mycoplasmatales|f__Mycoplasmataceae|g__Ureaplasma	k__Bacteria|p__Proteobacteria|c__Betaproteobacteria|o__Burkholderiales	0.22384	0.913407	0.178571	8.94768e-07	0.178571
k__Bacteria|p__Tenericutes|c__Mollicutes|o__Mycoplasmatales|f__Mycoplasmataceae|g__Ureaplasma	k__Bacteria|p__Firmicutes|c__Clostridia|o__Clostridiales|f__Clostridiales_Family_XI_Incertae_Sedis|g__Anaerococcus|s__Anaerococcus_tetradius	0.16908	0.91546	0.0952381	5.10511e-08	0.0952381
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales	k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales|f__Mycobacteriaceae|g__Mycobacterium	0.645521	0.770061	0.0238095	1.51737e-102	0.0238095
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales	k__Bacteria|p__Proteobacteria	0.47485	0.830855	0.0952381	3.97005e-66	0.0952381
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales	k__Bacteria|p__Proteobacteria|c__Betaproteobacteria|o__Burkholderiales	0.270275	0.895444	0.333333	6.96646e-14	0.333333
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales	k__Bacteria|p__Firmicutes|c__Clostridia|o__Clostridiales|f__Clostridiales_Family_XI_Incertae_Sedis|g__Anaerococcus|s__Anaerococcus_tetradius	0.177097	0.911452	0.214286	1.38729e-12	0.214286
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales	k__Bacteria|p__Firmicutes|c__Bacilli|o__Bacillales	0.11548	0.92714	0.345238	0.00333119	0.345238
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales|f__Mycobacteriaceae|g__Mycobacterium	k__Bacteria|p__Proteobacteria	0.247397	0.911876	0.47619	3.91306e-11	0.47619
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales|f__Mycobacteriaceae|g__Mycobacterium	k__Bacteria|p__Proteobacteria|c__Betaproteobacteria|o__Burkholderiales	0.21676	0.916146	0.214286	7.87945e-06	0.214286
k__Bacteria|p__Actinobacteria|c__Actinobacteria|o__Actinomycetales|f__Mycobacteriaceae|g__Mycobacterium	k__Bacteria|p__Firmicutes|c__Clostridia|o__Clostridiales|f__Clostridiales_Family_XI_Incertae_Sedis|g__Anaerococcus|s__Anaerococcus_tetradius	0.258307	0.870847	0.0714286	1.65264e-25	0.0714286
k__Bacteria|p__Proteobacteria	k__Bacteria|p__Proteobacteria|c__Betaproteobacteria|o__Burkholderiales	0.474772	0.816333	0.0238095	1.13119e-42	0.0238095
k__Bacteria|p__Proteobacteria	k__Bacteria|p__Firmicutes|c__Bacilli|o__Bacillales	0.125913	0.920558	0.214286	0.00366593	0.214286
k__Bacteria|p__Proteobacteria	k__Bacteria|p__Bacteroidetes|c__Bacteroidia|o__Bacteroidales|f__Bacteroidaceae|g__Bacteroides|s__Bacteroides_plebeius	0.139233	0.860767	0.0952381	4.18452e-06	0.0952381
k__Bacteria|p__Firmicutes|c__Bacilli|o__Bacillales	k__Bacteria|p__Bacteroidetes|c__Bacteroidia|o__Bacteroidales|f__Bacteroidaceae|g__Bacteroides|s__Bacteroides_plebeius	0.00114315	0.998857	0.047619	1	1
